html/main.js

```diff
@@ -212,25 +222,27 @@ function getTime(timing) {
 }
 
 function isError(timing) {
-    if (timing.success_percentage > 0) {
-        return false;
+    if (timing.success_percentage < 100) {
+        return 100 - timing.success_percentage;
     }
-    return (timing.error !== undefined);
+    if (timing.error !== undefined) {
+        return 100;
+    }
+    return 0;
 }
 
 const CompareFunctionCriterias = {
     'http.status_code': (a, b) => -compareTool(a, b, null, 'http', 'status_code'),
     'error': (a, b) => -compareTool(a, b, null, 'error'),
-    'error_wp': (a, b) => compareTool(a, b, null, 'timing', 'search_wp', 'error'),
```

Comment:
Since a long time, there is no more test on the wikipedia engine : remove dead code.